R/utils.R

```diff
@@ -384,3 +384,11 @@ df_print <- function(x, ...) {
   print(x, ...)
   invisible(x)
 }
+
+is_testing <- function() {
+  if (is_bool(opt <- peek_option("rlang:::is_testing"))) {
+    return(opt)
+  }
+
+  is_installed("testthat") && testthat::is_testing()
```

Comment:
We normally just inline `testthat::is_testing()`; and then use `local_envvar()` to override in tests.

Reply:
Do you mean instead of the peek-option? The is-installed is needed right?

Reply:
Yeah, we normally just do:

```R
is_testing <- function() {
    identical(Sys.getenv("TESTTHAT"), "true")
}
```

and then override the `TESTTHAT` var.  Or are you saying that doesn't work because it breaks testthat itself?

Reply:
oh ok, got it! I'll try that.